src/isa/riscv64/system/priv.c

```diff
@@ -1326,6 +1330,10 @@ static inline void csr_write(word_t *dest, word_t src) {
       }
     }
   #endif //CONFIG_RV_SSDBLTRP
+    // mstatus.MPP cannot hold MODE_HS=2
```

Comment:
There is nothing named MODE_HS. 2 is just a reserved code for MPP

Reply:
Right. This is an internal name in NEMU. It is used everywhere.